src/actions/channelsActions.js

```diff
@@ -36,7 +36,7 @@ export const putChannel = (
     await getTokenApp("putChannel");
     new Promise((resolve, reject) =>
         client
-            .put(`/api/v1/channels`, {
```

Comment:
I have unified both operations in one endpoint since now it has no sense to have separated ones. We need to rollback this change for LC integration

Reply:
if for now this change is closed it is not necessary